Description: Stores SECRET_KEY in /tmp for tests
 As horizon plugins are importing openstack_dashboard.test.settings, they
 attempt to write to /usr/lib/python2.7/dist-packages/openstack_dashboard/test
 which is read only. This patch should fix the problem.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-09-29

--- horizon-10.0.0~rc2.orig/openstack_dashboard/test/settings.py
+++ horizon-10.0.0~rc2/openstack_dashboard/test/settings.py
@@ -34,7 +34,7 @@ STATIC_URL = '/static/'
 WEBROOT = '/'
 
 SECRET_KEY = secret_key.generate_or_read_from_file(
-    os.path.join(TEST_DIR, '.secret_key_store'))
+    os.path.join('/tmp', '.secret_key_store'))
 ROOT_URLCONF = 'openstack_dashboard.test.urls'
 
 TEMPLATES[0]['DIRS'] = [
